templates/kubernetes/what-is-kubernetes.html

```diff
@@ -143,12 +185,96 @@ <h2 class="u-sv3">Kubernetes advantages</h2>
   </div>
 </section>
 
-<section class="p-strip--light">
+<section class="p-strip--suru">
+  <div class="u-fixed-width">
+    <h2>Kubernetes resources</h2>
+    <div class="row p-divider u-no-padding--left u-no-padding--right is-dark u-equal-height">
+      <div class="col-3 p-divider__block">
+        <div class="p-heading-icon--muted">
```

Comment:
I think `p-heading-icon--muted` class has been deprecated. Could we update the [heading icon](https://vanillaframework.io/docs/patterns/heading-icon) component?